### DISCUSSION TITLE
Games disappear

### DISCUSSION THREAD

**Discussion by mgutz** [herbstluftwm/herbstluftwm#1489] [General]
Playing Dota 2 full screen, then switching to another tags, then going back to Dota 2 tag results in the game disappearing. Works in other tiling WM (i3, qtile).

Any workaround ideas? I'm guessing a rule might help, but the game disappears if I launch any other app like xprop.

**Comment by mgutz** [comment]
The window is being minimized automatically when it loses focus, it can be disabled within game's video settings. Here's how to do it manually from terminal

```
xdotool windowactivate `xdotool search --class Dota | tail -1`
```

**Reply by bew** [reply]
Oh, I think it would be nice to have a way to ignore minimize requests on demand (via a hlwm rule).
I'm pretty sure not all games allows you to disable this
